ghost/core/test/e2e-api/admin/settings.test.js

```
@@ -41,7 +42,7 @@ const matchSettingsArray = (length) => {
     return settingsArray;
 };
 
-describe('Settings API', function () {
+describe.only('Settings API', function () {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)